### PR TITLE
[CARBONDATA-3476] Fix Read time and scan time stats in executor log for filter query

### PR DESCRIPTION
**Pull request by vikramahuja1001**
Problem: Read time and scan time stats shown wrong in executor log for filter query

Root cause: Projection read time is added in scan time because of this scan time and read time is not correct in stats

Solution: Added projection read time for both measure and dimension column in read stats

 - [ ] Any interfaces changed? NA
 
 - [ ] Any backward compatibility impacted? NA
 
 - [ ] Document update required? NA

 - [ ] Testing done 
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by brijoobopanna** [comment]
add to whitelist

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3862/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/12135/

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/4067/

File: core/src/main/java/org/apache/carbondata/core/scan/scanner/impl/BlockletFilterScanner.java [hunk 1]
```diff
@@ -96,8 +96,8 @@ public BlockletFilterScanner(BlockExecutionInfo blockExecutionInfo,
    * @param rawBlockletColumnChunks block chunk holder which holds the data
    * @throws FilterUnsupportedException
    */
-  @Override
-  public BlockletScannedResult scanBlocklet(RawBlockletColumnChunks rawBlockletColumnChunks)
+  @Override public BlockletScannedResult scanBlocklet(
```
Comment (kunal642) [inline core/src/main/java/org/apache/carbondata/core/scan/scanner/impl/BlockletFilterScanner.java]: Please revert any unnecessary changes like this.

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.1/15/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.2, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/20/
